packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -24,8 +24,8 @@ const TabsPrimitive: Primitive<TabsProps, 'div'> = (
           </TabsItem>
         ))}
       </TabList>
-      {items?.map(({ value, content, isDisabled }) => (
-        <TabPanel key={value} value={value} isDisabled={isDisabled}>
+      {items?.map(({ value, content, isDisabled, ...rest }) => (
+        <TabPanel {...rest} key={value} value={value} isDisabled={isDisabled}>
```

Comment:
qq: are there any items in rest that we don't want to spread into TabPanel? Geniunely asking as I don't know. I've seen cases in the past where we have passed down properties that aren't appropriate, which can throw a react warning in the console.

Reply:
`rest` just contains `label: React.ReactNode` for the TabPanel, and for the TabList it just has `isDisabled: Boolean | undefined`. We may as well just pass them directly at that point if that's intended, right?

Reply:
If additional parameters are added to the parent interface they'll bleed through. Is that behavior what we want here?

Reply:
I removed this spread, wasn't needed so I think it's safer to just get rid of it.